### ISSUE TITLE
🐛 BUG: `wrangler delete` fails with "Raw mode is not supported"

### ISSUE THREAD

**Issue by rozenmd** [cloudflare/wrangler2#2056]
### What version of `Wrangler` are you using?

beta

### What operating system are you using?

No idea

### Describe the Bug

feature-request: either wrangler delete -y or support yes y | wrangler delete without getting Error: Raw mode is not supported on the current process.stdin, which Ink uses as input stream by default. 🙂

> fwiw I'm in a standard bash terminal in linux

**Comment by rozenmd** [comment]
See https://github.com/cloudflare/wrangler2/issues/1992

**Comment by rozenmd** [comment]
Confirmed this is fixed:
```
$ yes y | npx wrangler delete
 ⛅️ wrangler 2.7.1 (update available 2.8.0)
-----------------------------------------------------
? Are you sure you want to delete basic-worker? This action cannot be undone.
🤖 Using default value in non-interactive context: yes
Successfully deleted basic-worker
```
